src/encoded/static/components/biosample.js

```diff
@@ -706,43 +706,56 @@ CharacterizationCaption.propTypes = {
 
 // Document detail component -- default
 const CharacterizationDetail = (props) => {
-    const doc = props.doc;
+    const characterization = props.doc;
     const keyClass = `document__detail${props.detailOpen ? ' active' : ''}`;
-    const excerpt = doc.description && doc.description.length > EXCERPT_LENGTH;
+    const excerpt = characterization.caption && characterization.caption.length > EXCERPT_LENGTH;
+
+    // See if we need a list of documents or not. Documents without attachments don't get
+    // displayed.
+    const docs = characterization.documents && characterization.documents.length ?
+        characterization.documents.filter(doc => !!(doc.attachment && doc.attachment.href && doc.attachment.download))
```

Comment:
`!!expr` can be written as `Boolean(expr)` which I find more readable.

Here's the relevant eslint rule if we want to check for this sort of thing: https://eslint.org/docs/rules/no-implicit-coercion